app/services/quantified_events/create_or_update_service.rb

```diff
@@ -30,7 +30,8 @@ def matching_billable_metric?
     delegate :customer, :subscription, :organization, to: :event
 
     def event_operation_type
-      event.properties['operation_type']&.to_sym
+      operation_type = event.properties['operation_type']&.to_sym
+      operation_type.nil? && matching_billable_metric&.unique_count_agg? ? :add : operation_type
```

Comment:
Style/TernaryParentheses: Use parentheses for ternary expressions with complex conditions.